repo/modules.go

```diff
@@ -94,17 +95,21 @@ func importRepoRulesModules(filename string, _ *RemoteCache) (repos []Repo, err
 		if mod, ok := pathToModule[path]; ok {
 			mod.Sum = sum
 		}
+		if mod, ok := replacePathToModule[path]; ok {
+			mod.Sum = sum
+		}
 	}
 
 	// If sums are missing, run go mod download to get them.
 	var missingSumArgs []string
 	for _, mod := range pathToModule {
 		if mod.Sum == "" {
-			if mod.Replace != nil {
-				missingSumArgs = append(missingSumArgs, fmt.Sprintf("%s@%s", mod.Replace.Path, mod.Replace.Version))
-			} else {
-				missingSumArgs = append(missingSumArgs, fmt.Sprintf("%s@%s", mod.Path, mod.Version))
-			}
+			missingSumArgs = append(missingSumArgs, fmt.Sprintf("%s@%s", mod.Path, mod.Version))
```

Comment:
If `pathToModule` is fixed as above, we shouldn't need the change below this point. We don't need sums for modules that were replaced. We especially don't need `go_repository` rules for both replaced and replacement modules.